test/integration/services/patient-quick-search.js

```diff
@@ -8,11 +8,12 @@ context('Patient Quick Search', function() {
       patient.id = `${ index }`;
       patient.first_name = 'Test';
       patient.last_name = `${ index } Patient`;
+      patient.identifiers = [];
```

Comment:
Still need to add tests here. But this change makes it so the test passes. Was failing without this.